src/NationalInstruments.Analyzers/Properties/Resources.resx

```diff
@@ -414,6 +414,16 @@
   <data name="NI1816X_Title" xml:space="preserve">
     <value>Disposable should not have finalizer</value>
   </data>
+  <data name="NI1019_Description" xml:space="preserve">
+    <value>The default implicit equality method implementation for records is to compare enumerable types by reference, not the element values;  thus it is important for records with enumerable properties to explicitly implement equality and compare the enumerables using SequenceEqual or UnorderedEquals (depending on whether ordering matters).</value>
+  </data>
+  <data name="NI1019_Message" xml:space="preserve">
+    <value>Record '{0}' has properties that are enumerable collections, but does not explicitly implement the Equals and GetHashCode methods</value>
```

Comment:
- Either remove the comma or add the subject to make it an independent clause
- I would explicitly say which Equals method because my mind would immediately think `override Equals(object other)`

"Record '{0}' has properties that are enumerable collections, but it does not explicitly implement the Equals(TRecord other) and GetHashCode methods"

Reply:
Sounds good!  I've updated it.